Modules/Devices/src/Devices.Domain/Aggregates/PushNotifications/PnsRegistration.cs

```diff
@@ -20,11 +21,11 @@ public PnsRegistration(IdentityAddress identityAddress, DeviceId deviceId, PnsHa
         Environment = environment;
     }
 
-    public IdentityAddress IdentityAddress { get; }
-    public DeviceId DeviceId { get; }
-    public DevicePushIdentifier DevicePushIdentifier { get; private set; }
-    public PnsHandle Handle { get; private set; }
-    public string AppId { get; private set; }
+    public IdentityAddress IdentityAddress { get; } = null!;
+    public DeviceId DeviceId { get; } = null!;
+    public DevicePushIdentifier DevicePushIdentifier { get; private set; } = null!;
+    public PnsHandle Handle { get; private set; } = null!;
+    public string AppId { get; private set; } = null!;
```

Comment:
What's that? Why do you assign null here to all of them?

Reply:
The empty constructor is making a big mess here, since because of it each property can be null. However, that constructor is only in place for the EF Core usage, and in fact none of the properties are ever null, they are set on instantiation. That is the reason why suppress the warnings in such a way and once all the properties are taken care of in this way... (continued int he next issue)